reqs/README.md

```diff
@@ -24,7 +23,6 @@ Requirements
     - reqs/requirements-dev-lite.txt
     - reqs/requirements-dev-contrib.txt
     - reqs/requirements-dev-api-docs-test.txt
-    - reqs/requirements-dev-cloud.txt
```

Comment:
This file is not currently in `requirements-dev-test.txt` and should be deleted from here. Thanks!

Reply:
ty - kept it removed